runtests

```diff
@@ -28,9 +28,12 @@ main() {
 
   echo 'Running shellcheck'
   git grep -l '^#!/usr/bin/env bash' | xargs shellcheck
+  # Temporarily disable shellcheck checks for SC2154 (undefined variable) on /bin/bash files
+  git grep -l '^#!/bin/bash' | xargs shellcheck -e SC2154
```

Comment:
Nice catch @edmorley, thanks!

Btw, I didn't include `/bin/sh` because `git grep` returns exit code 1 if it doesn't find any matches, so the script aborts because of `set -o errexit`. (We could pass `--no-run-if-empty` to `xargs` but it will still fail because of `set -o pipefail`).

Feel free to share a graceful solution if you're so inclined :)